ci/check_lerna_packages.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+HASH=$(git merge-base HEAD master)
+yarn lerna ls --since ${HASH} --include-dependents > lerna_output
+if [[ $(cat lerna_output | grep @) ]]; then
+  cat lerna_output | grep @ > lerna_packages
```

Comment:
Why do you need to grep for `@`?

Reply:
I guess it's the symbol that precedes the lerna package? eg `@uma/core`